stetho/src/main/java/com/facebook/stetho/common/android/ViewGroupUtil.java

```diff
@@ -0,0 +1,57 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+package com.facebook.stetho.common.android;
+
+import android.view.View;
+import android.view.ViewGroup;
+
+import java.lang.reflect.Field;
+
+import javax.annotation.Nullable;
+
+public final class ViewGroupUtil {
+  private static final Field sOnHierarchyChangeListenerField;
+  static {
+    Field field;
+    try {
+      field = ViewGroup.class.getDeclaredField("mOnHierarchyChangeListener");
+    } catch (NoSuchFieldException ex) {
+      field = null;
+    }
+
+    sOnHierarchyChangeListenerField = field;
+
+    if (sOnHierarchyChangeListenerField != null) {
+      sOnHierarchyChangeListenerField.setAccessible(true);
```

Comment:
To me, this reads more clearly as:

  try {
    Field field = ...;
    field.setAccessible(true);
    sOnHierarchyChangeListenerField = field;
  } catch (...) {
    sOnHierarchyChangeListenerField = null;
  }